tests/bsim/bluetooth/host/misc/acl_tx_frag/test_scripts/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+test_name="$(guess_test_long_name)"
+simulation_id=${test_name}
+verbosity_level=2
+
+# sixty-second (maximum) sim time.
+# The test will exit simulation as soon as it has passed.
+SIM_LEN_US=$((60 * 1000 * 1000))
+
+test_exe="${BSIM_OUT_PATH}/bin/bs_${BOARD_TS}_${test_name}_prj_conf"
+
+cd ${BSIM_OUT_PATH}/bin
+
+Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=0 -rs=420 -testid=dut \
+		-argstest log_level 3
+Execute "${test_exe}" -v=${verbosity_level} -s=${simulation_id} -d=1 -rs=69  -testid=peer \
+		-argstest log_level 3 >/dev/null
```

Comment:
> `>/dev/null`

was this left due to an oversight? (note you should be able to set the verbosity to 0 and get nothing but warnings out of a device)

Reply:
Oversight at first, but then realized logs from the other device are not useful. So I left it there.
I'll use the `-v` switch next time, sorry.

Reply:
nothing to be sorry about